examples/ComputeBoids.cpp

```diff
@@ -88,7 +88,7 @@ void initBuffers() {
             .GetResult();
 
         particleBuffers[i].SetSubData(0,
-            sizeof(Particle) * kNumParticles / sizeof(uint32_t),
+            sizeof(Particle) * kNumParticles,
             reinterpret_cast<uint32_t*>(initialParticles.data()));
```

Comment:
Can we have the `const uint32_t*` changed to a `const uint8_t*` to be consistent with the size argument?

Reply:
This will also make the wire work correctly, right now it thinks size is in number of uint32_t

Reply:
Done.